theatre/studio/src/panels/SequenceEditorPanel/AGGREGATE_COPY_PASTE.md

```diff
@@ -0,0 +1,24 @@
+copy algorithm: find the closest common acnestor for the tracks selected
+
+- obj1.props.transform.position.x => simple
+- obj1.props.transform.position.{x, z} => {x, z}
+- obj1.props.transform.position.{x, z} + obj1.props.transform.rotation.z =>
+  {position: {x, z}, rotation: {z}}
```

Comment:
This would be nice but I haven't yet become acquainted with our tests. I would also have to pull the copy paste functionality up and out of where it is or do some heavy mocking to make this work. I'm not sure what approach would be practical/useful here.